scale-tests/deploy-dispatchers.py

```diff
@@ -0,0 +1,58 @@
+import shakedown
+import sys
+
+
+# This script will deploy the specified number of dispatchers with an optional
+# options json file. It will take the given base service name and
+# append an index to generate a unique service name for each dispatcher.
+#
+# The service names of the deployed dispatchers will be written into an output
+# file.
+#
+# Running:
+# > dcos cluster setup <cluster url>
+# > python deploy-dispatchers.py 2 spark-instance dispatchers.out
+
+
+SPARK_PACKAGE_NAME="spark"
+
+
+#TODO: add options json file contents
+def deploy_dispatchers(num_dispatchers, service_name_base, output_file, options_json_file):
+    outfile = open(output_file, "w")
```

Comment:
This outfile is unused since you declare a new one on L24

Reply:
Good catch. I'll delete this line.